lib/node_modules/@stdlib/blas/ext/base/drssors/test/test.drssors.js

```diff
@@ -0,0 +1,212 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var tape = require( 'tape' );
+var Float64Array = require( '@stdlib/array/float64' );
+var ulpdiff = require( '@stdlib/number/float64/base/ulp-difference' );
+var drssors = require( './../lib/drssors.js' );
+
+
+// TESTS //
+
+tape( 'main export is a function', function test( t ) {
+	t.ok( true, __filename );
+	t.strictEqual( typeof drssors, 'function', 'main export is a function' );
+	t.end();
+});
+
+tape( 'the function has an arity of 5', function test( t ) {
+	t.strictEqual( drssors.length, 5, 'returns expected value' );
+	t.end();
+});
+
+tape( 'the function calculates the residual sum of squares (ignoring NaN values)', function test( t ) {
+	var x;
+	var y;
+	var z;
+
+	x = new Float64Array( [ 1.0, -2.0, -4.0, 5.0, 0.0, 3.0 ] );
```

Comment:
Your implementation is designed to ignore NaN values, and yet your tests do not test for this behavior at all. They all contain only non-NaN values. You should study `stats/strided` for how we test against `nan*` and non-nan variants.